testgrid/config.yaml

```diff
@@ -5787,6 +5789,11 @@ dashboards:
     test_group_name: pull-aws-ebs-csi-driver-sanity
     description: aws ebs csi driver sanity test
 
+- name: sig-aws-eks-pull-kubernetes-e2e-1-10
```

Comment:
Maybe make the dashboard name more generic - like sig-aws-eks-presubmits or sth?